src/interface/TWCoinTypeConfiguration.cpp

```diff
@@ -304,4 +304,48 @@ TWString *_Nonnull TWCoinTypeConfigurationGetName(enum TWCoinType type) {
     return TWStringCreateWithUTF8Bytes(string.c_str());
 }
 
+TWString *_Nullable TWCoinTypeConfigurationGetNodeSource(enum TWCoinType type) {
+    string string;
+    switch (type) {
+    case TWCoinTypeBitcoin:
```

Comment:
Attention. For all cases below result will be: "https://github.com/trezor/blockbook"
Need add break or empty string